chsdi/models/vector/__init__.py

```diff
@@ -86,7 +86,8 @@ def __read__(self):
                             len(self._shape) < MAX_FEATURE_GEOMETRY_SIZE:
                         geom = self._shape
                     elif val is not None and \
-                            len(val.data) < MAX_FEATURE_GEOMETRY_SIZE:
+                            (len(val.data) < MAX_FEATURE_GEOMETRY_SIZE or
+                            self.__ignore_max_feature_geometry_size__):
```

Comment:
add `__ignore_max_feature_geometry_size__` to the base Vector model. If not defined, you get an error.